airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGeneratorIntegrationTest.java

```diff
@@ -308,10 +314,7 @@ SELECT _airbyte_ab_id, _airbyte_data, _airbyte_emitted_at FROM UNNEST([
   @Override
   protected List<JsonNode> dumpRawTableRecords(final StreamId streamId) throws Exception {
     final TableResult result = bq.query(QueryJobConfiguration.of("SELECT * FROM " + streamId.rawTableId(BigQuerySqlGenerator.QUOTE)));
-    return BigQuerySqlGeneratorIntegrationTest.toJsonRecords(result).stream().peek(record -> {
```

Comment:
this logic already lives in the RecordDiffer, so no need to explicitly deserialize here.